src/utils/jira/JiraTicketStatusFetcher.php

```diff
@@ -57,24 +57,24 @@ private static function decodeAndValidateResponse(string $body): array
         $data = json_decode($body, true);
 
         if (!is_array($data) || !array_key_exists('fields', $data)) {
-            throw self::throwInvalidResponse();
```

Comment:
opened a phpstan issue on that one, as it should be detected as a error:

https://github.com/phpstan/phpstan/issues/10344